GameEngine/GameState.cs

```diff
@@ -24,27 +24,27 @@ public GameState(IPlayer player, Deck deck, int gameSizeMultiplier)
         public void StartGame()
         {
             var hand = Deck.Draw(3);
-            Player.AddCardsToHand(hand);
+            Player.Hand.Add(hand);
         }
 
         public void TakeTurn()
         {
             Console.WriteLine("Taking a turn");
-            if (Player.HandContainsSun())
+            if (Player.Hand.ContainsSun)
```

Comment:
💡 Light preference for `Hand.Contains(CardType.Sun)`, but it's also a bit of future-proofing.

Reply:
Nothing stopping you from using `Hand.Cards.Contains(CardType.ENUM)`, this was just a shortcut for Sun, being the special case.